diff --git a/bin/Data/Scripts/Editor.as b/bin/Data/Scripts/Editor.as
index cb92907..b835be8 100644
--- a/bin/Data/Scripts/Editor.as
+++ b/bin/Data/Scripts/Editor.as
@@ -22,6 +22,7 @@
 #include "Scripts/Editor/EditorColorWheel.as"
 #include "Scripts/Editor/EditorEventsHandlers.as"
 #include "Scripts/Editor/EditorViewDebugIcons.as"
+#include "MyData/Scripts/EditorExt.as"
 
 String configFileName;
 
@@ -146,6 +147,8 @@ void HandleUpdate(StringHash eventType, VariantMap& eventData)
             }
         }
     }
+
+    UpdateExt(timeStep);
 }
 
 void HandleReloadFinishOrFail(StringHash eventType, VariantMap& eventData)
@@ -181,7 +184,7 @@ void LoadConfig()
     XMLElement soundTypesElem = configElem.GetChild("soundtypes");
     XMLElement cubeMapElem = configElem.GetChild("cubegen");
     XMLElement defaultTagsElem = configElem.GetChild("tags");
-    
+
     if (!cameraElem.isNull)
     {
         if (cameraElem.HasAttribute("nearclip")) viewNearClip = cameraElem.GetFloat("nearclip");
@@ -313,7 +316,7 @@ void LoadConfig()
         cubeMapGen_Path = cubemapDefaultOutputPath;
         cubeMapGen_Size = 128;
     }
-    
+
     if (!defaultTagsElem.isNull)
     {
         if (defaultTagsElem.HasAttribute("tags")) defaultTags = defaultTagsElem.GetAttribute("tags");
@@ -414,13 +417,13 @@ void SaveConfig()
     consoleElem.SetAttribute("commandinterpreter", console.commandInterpreter);
 
     varNamesElem.SetVariantMap(globalVarNames);
-    
+
     cubeGenElem.SetAttribute("name", cubeMapGen_Name);
     cubeGenElem.SetAttribute("path", cubeMapGen_Path);
     cubeGenElem.SetAttribute("size", cubeMapGen_Size);
 
     defaultTagsElem.SetAttribute("tags", defaultTags);
-    
+
     SaveSoundTypes(soundTypesElem);
 
     config.Save(File(configFileName, FILE_WRITE));
